src/main/java/com/kc/project/rashmiwebservice/RashmiSet.java

```diff
@@ -20,4 +20,25 @@ public void add(int item){
             intArr = tempArr;
         }
     }
+
+    public void addUnique(int item){
+        boolean dulpicate=false;
+        for(int j=0;j<intArr.length;j++){
+            if(intArr[j]==item){
+                dulpicate=true;
```

Comment:
Another option here is to just 'return;' then you can remove the 'if(!duplicate)' logic. 

fewer lines of code that way :-)